src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            List<String> linesFromFile = Files.readAllLines(Path.of(fromFileName));
+            LinkedHashMap<String, Integer> linkedHashMap = new LinkedHashMap<>();
+            for (String line : linesFromFile) {
```

Comment:
Instead of using a LinkedHashMap to store the statistics, consider defining a simple class or a record to hold the supply and buy amounts. This will make the code more readable and less error-prone.